torchgen/gen.py

```diff
@@ -2321,6 +2335,9 @@ def gen_op_headers(
             "CompositeViewCopyKernel_Definitions": list(
                 mapMaybe(gen_composite_view_copy_kernel, view_groups)
             ),
+            "SymIntViewCopyKernel_Definitions": list(
+                mapMaybe(lambda pair: gen_symint_view_copy_kernel(pair[0], pair[1]), view_copy_with_symint_pairs)
```

Comment:
cc @ezyang  I remember hearing that long term we'd like to have `view*.SymInt` fully subsume the existing view/view copy ops, so we can always rip this out later.

But for now, I'm codegen'ing `{view}_copy.SymInt` kernel overloads to call into their `{view}_copy` variants, which is what the existing `expand_copy.SymInt` kernel does today.